lib/src/internal/interaction_endpoints.dart

```diff
@@ -276,8 +276,9 @@ class InteractionsEndpoints implements IInteractionsEndpoints {
   }
 
   @override
-  Stream<ISlashCommand> fetchGlobalCommands(Snowflake applicationId) async* {
-    final response = await _client.httpEndpoints.sendRawRequest("/applications/$applicationId/commands", "GET", auth: true);
+  Stream<ISlashCommand> fetchGlobalCommands(Snowflake applicationId, {bool withLocales = false}) async* {
+    final Map<String, dynamic> queryParams = withLocales ? {'with_locales': 'true'} : {};
```

Comment:
Could you embed this in the line below like you did for `fetchGuildCommand`? Also, it should be `with_localizations` and not `with_locales`. Also no need for the ternary, `withLocales.toString()` is fine,